lib/sensors/accelerometer.hpp

```diff
@@ -50,24 +49,40 @@ constexpr std::uint8_t kExpectedDeviceIdValue = 0x44;
 
 class Accelerometer : public II2cMuxSensor<core::RawAccelerationData> {
  public:
-  Accelerometer(core::ILogger &logger, io::II2c &i2c, const std::uint8_t channel);
+  static std::optional<Accelerometer> create(core::ILogger &logger,
+                                             std::shared_ptr<io::II2c> i2c,
+                                             const std::uint8_t channel,
+                                             const std::uint8_t device_address);
```

Comment:
I think that here we want to make the "device address" an optional parameter, since in the implementation used oustide of the repl we don't care about the address.

Reply:
We are changing this so now we must be explicit about all parameters, i.e. no default values